procrastinate/contrib/django/admin.py

```diff
@@ -106,12 +119,19 @@ def short_task_name(self, instance: models.ProcrastinateJob) -> str:
 
     @admin.display(description="Args")
     def pretty_args(self, instance: models.ProcrastinateJob) -> str:
-        indent = 2 if len(instance.args) > 1 or len(str(instance.args)) > 30 else None
-        pretty_json = json.dumps(instance.args, indent=indent)
-        if len(pretty_json) > 2000:
-            pretty_json = pretty_json[:2000] + "..."
+        arg_rows = [
+            format_html(
+                "<tr><td>{key}</td><td>{value}</td></tr>",
+                key=key,
+                value=ellipsize_middle(value),
+            )
+            for key, value in instance.args.items()
+        ]
         return format_html(
```

Comment:
I could be wrong, but this looks like [format_html_join](https://docs.djangoproject.com/en/5.2/ref/utils/#django.utils.html.format_html_join) in disguise ?